uwblueprint/community-fridge-kw
# feat: Create scheduling get started screen

## Notion ticket link
[Scheduling Flow - Initial Scheduling Page]https://www.notion.so/uwblueprintexecs/Task-Board-db95cd7b93f245f78ee85e3a8a6a316d)](https://www.notion.so/uwblueprintexecs/Scheduling-Flow-Initial-Scheduling-Page-22820e0d32bf46659491cd5bbf52f5e3



## Implementation description
* Added text and button styling for Get Started page/step of scheduling process



## Steps to test
1. Make sure no errors and can run this page
2. Matches mobile designs in here: https://www.notion.so/uwblueprintexecs/Scheduling-Flow-Initial-Scheduling-Page-22820e0d32bf46659491cd5bbf52f5e3#c2282da7b22f439cace3d666fb2534a5


https://user-images.githubusercontent.com/37950626/141231193-01471f63-6792-4bac-8c95-454d13aa0611.mov


## What should reviewers focus on?
* 


## Checklist
